e2e-tests/Pipfile

```diff
@@ -0,0 +1,25 @@
+[[source]]
+
+url = "https://pypi.python.org/simple"
```

Comment:
Is this still the right url to use now that the new pypi is out? I know they're probably redirecting requests, but I don't know offhand if they have new official urls or not.

Reply:
Thanks for your review, approval, and asking about the canonical PyPi URL; no, it doesn't look that way, given https://packaging.python.org/guides/migrating-to-pypi-org/#publishing-releases, which says:

```As of April 13th, 2018,“pypi.org“ is the URL for PyPI.
```

But Kenneth says "The old version will continue to work for a long time.": https://github.com/pypa/pipenv/pull/1668#issuecomment-372259980

Reply:
I've brought it back, and -- surprise! -- we're passing again:

https://qa-preprod-master.fxtest.jenkins.stage.mozaws.net/job/socorro.adhoc/214/